tests/conftest.py

```diff
@@ -76,6 +79,11 @@ def java_args() -> Tuple[str]:
     return cast(Tuple[str], ())
 
 
+@fixture
+def profiler_state(tmp_path_world_accessible: Path) -> ProfilerState:
```

Comment:
Why `tmp_path_world_accessible`? I see that previously, some tests used just `tmp_path` for this purpose. Or some used `tmp_path` and some used `tmp_path_world_accessible`?

Please explain why you use `tmp_path_world_accessible` here in a comment :+1:

Reply:
I used it because some tests in `test_java` used `tmp_path_world_accessible`. I will check if it works with `tmp_path` instead.

Reply:
I changed it to `tmp_path` here and just keep requesting `tmp_path_world_accessible` in tests that were using it.

Reply:
Fair enough